constants/Backend.js

```diff
@@ -586,8 +586,12 @@ export const api = {
     getReservationByMonth: (year, month) => {
       return `${apiRoot}/reservations/byMonth?yearMonth=${year}-${month}`
     },
-    getAvailableTables: (date) => {
-      return `${apiRoot}/reservations/availableTables?reservationDate=${date}`
+    getAvailableTables: (date, id) => {
+      let params = ''
+      if (id != null) {
+        params += `&reservationId=${id}`
+      }
+      return `${apiRoot}/reservations/availableTables?reservationDate=${date}${params}`
```

Comment:
I would consider rewriting the logic like this:

let url = `${apiRoot}/reservations/availableTables?reservationDate=${date}`

if (id != null) {
  url += `&reservationId=${id}`
}
return url

IMO (in my opinion) the difference is that the variable content in the second style is more consistent.